dotcom-rendering/src/web/components/FrontCard.tsx

```diff
@@ -4,7 +4,26 @@ import { Card } from './Card/Card';
 
 type Props = {
 	trail: TrailType;
-} & Partial<CardProps>;
+	headlineSize?: SmallHeadlineSize;
+	showQuotes?: boolean;
+	showByline?: boolean;
```

Comment:
I think we can remove these. We probably only ever want to respect the values set by the tool. If this changes later we can add them in